contrib/devtools/update-translations.py

```diff
@@ -122,6 +122,12 @@ def escape_cdata(text):
     text = text.replace('"', '&quot;')
     return text
 
+def check_bitcoin_addr(text):
+    p = re.compile('^([13]|bc1)[a-zA-Z0-9]{33,}$')
```

Comment:
Probably want to make this a constant computed once on load rather than once per method call.